pages/1.13/release-notes/1.13.0-beta1/index.md

```diff
@@ -0,0 +1,295 @@
+
+- Dynamic masters on the Universal Installer. <!--(DCOS-45725)-->
+
+    Dynamic masters is the ability to create, destroy and recover master nodes. This enables users on the Universal Installer to be able to downscale or upscale their clusters from not just the agent nodes (which is currently supported), but also from the master nodes as well - if they deem necessary.
+
+- Universal Installer and on-premise DC/OS life cycle management with Ansible. <!--(DCOS-45724)-->
+
+    dcos-ansible is a Mesosphere provided and supported ansible roles for installing and upgrading Mesosphere DC/OS and DC/OS Enterprise on any user provided infrastructure.
+
+<!-- not in 1.13 Docs with RN filter 
+- Universal Installer to provision EBS volumes (DCOS-47221)
+The Universal Installer now provides the ability to provision AWS EBS volumes and attach them to the private agents within a cluster.
+
+Documented here: https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/install/provision-extra-volumes/
+-->
+
+## Networking
+- Add a new networking API endpoint to retrieve the public-facing IP address for public agent nodes. <!--(DCOS-28127)-->
+
+    This release introduces a new API endpoint for accessing public-facing IP addresses for the nodes in a cluster. For more information about retrieving and viewing public IP addresses, see [Finding the public IP address](/1.13/administering-clusters/locate-public-agent/).
+    
+    You can look up the public agent IP address using the DC/OS web-based console, command-line interface, or API calls for DC/OS cluster nodes if DC/OS is deployed on a public cloud provider such as AWS, Google Cloud, or Azure. If DC/OS is installed on an internal network (on-premise) or a private cloud, nodes do not typically have separate public and private IP addresses. For nodes on an internal network or private cloud, the public IP address is most often the same as the IP address defined for the server in the DNS namespace.
+
+<!-- not in 1.13 Docs in RN filter 
+- Retention policies for dcos-monitoring data (DCOS-46818)
+The dcos-monitoring service in versions 0.4.3 and later provides the ability to adjust the retention period of the Prometheus time series database. For more information see: https://docs.mesosphere.com/services/beta-dcos-monitoring/0.4.3-beta/operations/prometheus/storage/
+-->
+<!-- not in 1.13 Docs in RN filter 
+- UCR to support Schema2_v2 formatted docker images (DCOS-43871)
+Universal Container Runtime (UCR) now fully supports Docker Image format v2_schema2 in addition to supporting v2_schema1
+-->
+<!-- not in 1.13 Docs in RN filter 
+- Display Grafana dashboards on unsupervised displays (DCOS-51133)
+dcos-monitoring now enables Grafana dashboards to be displayed on read-only devices such as SmartTVs, kiosks or public panels.
+-->
+
+## Storage
+- Update Rex-Ray to support NVMe EBS volumes. <!--(DCOS-50047)-->
```

Comment:
please change the heading to be "Update Beta REX-Ray to support NVMe EBS volumes"

Reply:
change.